Source/WebCore/editing/TextIterator.cpp

```diff
@@ -2343,7 +2345,7 @@ inline bool SearchBuffer::atBreak() const
 
 inline void SearchBuffer::append(UChar c, bool isStart)
 {
-    m_buffer[m_cursor] = c == noBreakSpace ? ' ' : foldQuoteMark(c);
+    m_buffer[m_cursor] = foldQuoteMarkAndReplaceNoBreakSpace(c);
```

Comment:
Nothing here indicates this code was specific to macOS??

Reply:
Ah, I must be misremembering — for some reason, I had thought that the `UCONFIG_NO_COLLATION` flag differed between macOS and iOS. I'll fix the misleading commit message...

Reply:
Confirmed — both macOS and iOS build with `!UCONFIG_NO_COLLATION`. I'm not sure where I got the impression that they were different 🤔... Maybe at one point in time, that was the case.

I've updated the commit message to correct this.